tests/src/main/routes/change-password-routes.test.ts

```diff
@@ -0,0 +1,100 @@
+import app from '@/main/config/app'
+import { PrismaHelper } from '@/tests/helpers/prisma-helper'
+import request from 'supertest'
+
+beforeEach(async () => { await PrismaHelper.connect() })
+
+afterEach(async () => { await PrismaHelper.disconnect() })
+
+const makeSetup = async (): Promise<{ accessToken: string }> => {
+  await request(app)
+    .post('/api/signup')
```

Comment:
I have a propose to integration tests. Order tests and tell a 'story', maybe test User Story, not a route, make sense? Example: all tests that we need a guardian in db, we execute a POST /api/signup. This will turn tests more efficiently.